h/templates/accounts/base.html.jinja2

```diff
@@ -0,0 +1,33 @@
+{% extends "h:templates/layouts/base.html.jinja2" %}
+
+{% block header %}
+  {% include "h:templates/includes/logo-header.html.jinja2" %}
+{% endblock %}
+
+{% block content %}
+  <div class="form-container content">
+    {% include "h:templates/includes/flash-messages.html.jinja2" %}
+
+    <h1 class="form-header">
+      {% block page_title %}{% endblock %}
+    </h1>
+
+    <div class="form-description">
+      {% block form_message %}{% endblock %}
+    </div>
+
+    {% if form %}<p>{{ form }}</p>{% endif %}
+
+    <footer class="form-footer">
+      {% block form_footer %}{% endblock %}
+      {% if signup_message %}
+        {{ signup_message }}
+        <a class="link--footer" href="{{ request.route_path('signup') }}">Sign up</a>
+      {% endif %}
+      {% if login_message %}
+        {{ login_message }}
+        <a class="link--footer" href="{{ request.route_path('login') }}">Log in</a>
+      {% endif %}
```

Comment:
This was kind of bugging me, as these are both basically the same thing. Honestly though there's no way of doing it that's much shorter.

One idea for the future is you could add a "link-footer" macro in this template which does this. This would make this block just like those above, but child templates could do something like:

```
{% block form_footer %}{{ footer_link(message='Do blah', type='sign-up') }}{% endblock %}
```

The benefit being, the footer _could_ contain anything. No point here though, not enough variation to make it worth while.